scripts/push-documentation.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+setup_git() {
+  git config --global user.email "nakul.korde@intel.com"
+  git config --global user.name "Nakul Korde"
+}
+
+commit_website_files() {
+  mkdir upload_docs
+  cd upload_docs
+  git clone https://github.com/OPAE/opae.github.io.git
+  cd doc-testing
```

Comment:
Not sure how this works - doesn't the `git clone` in line 11 create a directory called "opae.github.io"? Where does `doc-testing` come from?

Reply:
How and where is the directory that's named after the tag created?

Reply:
Yes you are right. It should create a directory opae.github.io and not doc-testing. It was a part of the testing I was doing and pushed the wrong version of the script. I have corrected the error now.

When the documentation is built, it would create a directory for the html files named with the current release in the location as can be seen from here: mybuild_docs/sphinx/html/

https://github.com/OPAE/opae-sdk/blob/55d987f5863ceb3e7f781ddb306409fa190ffe48/doc/sphinx/CMakeLists.txt#L42-L43

I then copy that directory to the where I have cloned the opae.github.io repo and then push it to opae.github.io.